multi_import/fields.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import relations
+
+from multi_import.utils import normalize_string
+
+__all__ = [
+    'FieldHelper',
+]
+
+
+class FieldHelper(object):
+    list_separator = ';'
+
+    def to_string_representation(self, field, value):
+        if hasattr(field, 'to_string_representation'):
```

Comment:
I've made it optional, whereas before I had it as mandatory. For the time-being I think that it's an ok compromise.

The consumer would need to have knowledge on a per field-basis how to convert to/from a string - so I'm not entirely convinced that it should be the responsibility of the consumer.